auxiliary/notes.tex

```diff
@@ -4,11 +4,42 @@
 This version of the book has been substantially revised
 since the first release in June 2019
 with feedback from readers and other experts.
```

Comment:
I suggest moving this below the acknowledgments, to come right after the sentence about DIME team. The current flow is a little strange (note on revisions, acknowledgments, more on revision).